models/_root/settings.yml

```diff
@@ -22,42 +22,42 @@ properties:
           properties:
             annotation:
               type: string
-              description: String containing HTML attributes associated with this button
+              description: String containing HTML attributes associated with this button.
             service:
               type: string
               description: 'String containing the name of this button''s social-media service (facebook, email, print, twitter, linkedin, google, etc.)'
   address:
     type: string
-    description: 'Store’s complete physical address, as entered by merchant in the Store Profile'
-  amp_analytics_id:
-    type: string
-    description: Returns Google AMP analytics ID
+    description: 'Store’s complete physical address, as entered by merchant in the Store Profile.'
   base_url:
     type: string
-    description: The normal shop URL
+    description: The normal shop URL.
   blog_enabled:
     type: boolean
-    description: Site-wide boolean value that indicates whether the blog is visible for this store
+    description: Site-wide boolean value that indicates whether the blog is visible for this store.
   bulk_discount_enabled:
     type: boolean
     description: Boolean that return true if bulk discount is enabled on products.
   cdn_url:
     type: string
   client_ip_address:
     type: string
-    description: IP address of the customer browsing the store
+    description: IP address of the customer browsing the store.
   country_code:
     type: string
-    description: The country code corresponding to the IP
+    description: The country code corresponding to the IP.
   data_tag_enabled:
     type: boolean
-    description: 'Site-wide boolean that indicates whether GAEE is enabled in a theme. For {{settings.data_tag_enabled}} to be true, the enhanced_ecommerce key must be present in config.json and a GAEE experiment must be enabled along with GA property value set in Advanced Settings > Analytics > GA in the BigCommerce Control Panel'
+    description: 'Site-wide boolean that indicates whether GAEE is enabled in a theme. For {{settings.data_tag_enabled}} to be true, the enhanced_ecommerce key must be present in config.json and a GAEE experiment must be enabled along with GA property value set in Advanced Settings > Analytics > GA in the BigCommerce control panel.'

```

Comment:
```suggestion
    description: 'Site-wide boolean that indicates whether GAEE is enabled in a theme. For {{settings.data_tag_enabled}} to be true, the enhanced_ecommerce key must be present in config.json and a GAEE experiment must be enabled along with GA property value set in Advanced Settings > Analytics > GA in the store control panel.'

```

Reply:
@bc-tgomez I wonder what happens to this with GA4 -- can you make a ticket for someone to investigate?

Reply:
Sure will.